installer.py

```diff
@@ -26,12 +28,16 @@
 def main():
     missing_utils = [u for u in REQUIRED_UTILS if not which(u)]
     if missing_utils:
-        raise RuntimeError("missing system utilities: %s" % ", ".join(missing_utils))
+        raise RuntimeError("Missing system utilities: %s" % ", ".join(missing_utils))
+
+    missing_libs = [lib for lib in REQUIRED_LIBS if lib not in installed_libs()]
+    if missing_libs:
+        raise RuntimeError("Missing system libraries: %s" % ", ".join(missing_libs))
 
     env_info = _get_env_info()
 
     if env_info.plat not in {PLAT_DARWIN, PLAT_LINUX} or platform.machine() != "x86_64":
-        raise RuntimeError("Unsupported platform: %s (%s)" % (env_info.plat, platform.machine()))
+        raise RuntimeError("Unsupported platform: %s (%s)." % (env_info.plat, platform.machine()))
```

Comment:
```suggestion
        raise RuntimeError("Unsupported platform: %s (%s)" % (env_info.plat, platform.machine()))
```
This is inconsistent with the other error messages. Please either make them all `.` (as I think you seem to prefer :) or all ``